src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1426,4 +1425,19 @@
     <value>Bold font with bright colors</value>
     <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will be rendered as both bold text and in a brighter color</comment>
   </data>
-</root>
+  <data name="Globals_ConsoleHostStartupMode.Header" xml:space="preserve">
+    <value>How to start when acting a the default console host</value>

```

Comment:
```suggestion
    <value>Default Terminal Launch Mode</value>
    <comment>Header for a control to select what mode to launch the terminal in when a new session is started by a commandline application. "Default Terminal" refers to the "default terminal application" setting.</comment>
```
The Header here is what's shown in the expander's header for the settings UI.
The comment can be changed to whatever. I just want to make sure the translation is accurate.